docs/Installing.asciidoc

```diff
@@ -99,10 +99,50 @@ You can do that in +/etc/openqa/openqa.ini+:
 httpsonly = 0
 --------------------------------------------------------------------------------
 
+=== Database engine (PostgreSQL)
+
+To setup openQA you will need a PostgreSQL database available to the user
```

Comment:
I wonder: the previous steps say zypper in openQA from the devel:openQA repo. That zypper command will already give you a working postgresql setup and start the postgresql.service. But adding this here makes the user assume he has to do these steps manually. 

Should we split this into 'from package' and 'from git'?